src/include/ipc/dai-intel.h

```diff
@@ -83,6 +83,10 @@ struct sof_ipc_dai_ssp_params {
 	uint16_t tdm_per_slot_padding_flag;
 	uint32_t clks_control;
 	uint32_t quirks;
+
+	/* M/N divider */
+	uint32_t divider_m;
```

Comment:
Do we need additional parameters? Can't we just calculate the m/n values based on the passed mclk and bclk?

Reply:
@tlauda We can (@abonislawski even did this before). It was request from @lbetlej to have it passed directly from topology (but I don't remember why).
@lbetlej Why do we want it from topology instead of computing it?

Reply:
@tlauda @jajanusz my recommendation was motivated to get m/n dividers control in place as fast as possible, if there is already code that is calculating optimal m/n divider settings depending on other settings defined in the topology file I have no objection to reuse it.

Reply:
@lbetlej oh ok, I'll change it to automatic then. It's way better solution.